HackIllinois/ViewControllers/HIProfileViewController.swift

```diff
@@ -125,228 +94,71 @@ extension HIProfileViewController {
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        scrollView.setContentOffset(.zero, animated: true)
         updateProfile()
         reloadProfile()
     }
 
     func updateProfile() {
         guard let profile = HIApplicationStateController.shared.profile else { return }
-        guard let user = HIApplicationStateController.shared.user else { return }
         view.layoutIfNeeded()
 
-        // Generate a QR Code with the user's id
-        DispatchQueue.global(qos: .userInitiated).async {
-            let qrImage: UIImage!
-            if let logo = UIImage(named: "QRCodeLogo"), let whiteBackground = UIImage(named: "QRCodeBackground") {
-                // Custom QR Code with logo
-                qrImage = UIImage.init(qrString: user.id, qrCodeColor: (\HIAppearance.qrTint).value, backgroundImage: whiteBackground, logo: logo)
-            } else {
-                // Regular, black QR Code
-                let size = max(UIScreen.main.bounds.width, UIScreen.main.bounds.height)
-                qrImage = UIImage.init(qrString: user.id, size: size)
-            }
-            DispatchQueue.main.async {
-                self.qrImageView.image = qrImage
-            }
-        }
+        if let url = URL(string: profile.avatarUrl), let imgValue = HIConstants.PROFILE_IMAGES[url.absoluteString] {
+                    profilePictureView.changeImage(newImage: imgValue)
+                }
         profileNameView.text = profile.firstName + " " + profile.lastName
-        let modifiedTeamStatus = profile.teamStatus.capitalized.replacingOccurrences(of: "_", with: " ")
-        if modifiedTeamStatus == "Looking For Team" {
-            profileStatusIndicator.changeCircleColor(color: \.groupSearchText)
-            profileStatusDescriptionView.changeTextColor(color: \.groupSearchText)
-        } else if modifiedTeamStatus == "Looking For Members" {
-            profileStatusIndicator.changeCircleColor(color: \.memberSearchText)
-            profileStatusDescriptionView.changeTextColor(color: \.memberSearchText)
-        } else {
-            profileStatusIndicator.changeCircleColor(color: \.noSearchText)
-            profileStatusDescriptionView.changeTextColor(color: \.noSearchText)
-        }
-        profileStatusDescriptionView.text = modifiedTeamStatus
-        profilePointsView.text = "\(profile.points)"
-        profileTimeView.text = profile.timezone
-        profileDescriptionView.text = profile.info
-        profileDiscordUsernameView.text = profile.discord
-        interests = profile.interests
-        profileInterestsView.reloadData()
-        profileInterestsViewHeight.constant = profileInterestsView.collectionViewLayout.collectionViewContentSize.height + 20
-
-        profileDiscordImageView.image = #imageLiteral(resourceName: "DiscordLogo")
-
-    }
+        profilePointsLabel.text = "\(profile.points)"
```

Comment:
Append the string "Points" after the value in the profilePointsLabel (example: "480 Points").